experiments/src/matcher.rs

```diff
@@ -25,7 +25,7 @@ pub struct Matcher {
 #[derive(Deserialize, Serialize, Debug, Clone, Default)]
 pub struct AppContext {
     pub app_id: Option<String>,
-    pub app_version: Option<String>,
+    pub app_version: Option<u32>,
```

Comment:
I am not sure if this will work, the app versions we have can be something like 79.4. I am also worried that they might also just be strings everywhere in the experimenter platform

Reply:
Ya that's a good point. I'll change it back for now! Transforms will make the comparison here possible with string versions, but we can worry about that later.